src/components/_chips.scss

```
@@ -0,0 +1,260 @@
+// Chips
```

```suggestion
// Chip
```
I think we can keep it singular. Since they can exist on their own

---

> When it's disabled the close arrow should not be reachable no? I can still hover it and click and my cursor change. Maybe you should remove the arrow button completely since it's disabled and no interaction can be done. 

Its a looks like a disabled state but for design wise its empty state so they can remove it instantly.

---

ha ok. in that case we should call it empty state? let's rename it to align with design wish.

---

I don't think we should call it as a empty state (since its a visual apperanace) in stylabilla, but in react component we should maintain the state as a empty state.